x-pack/plugins/maps/public/selectors/map_selectors.ts

```diff
@@ -432,7 +432,7 @@ export const areLayersLoaded = createSelector(
         layer.isVisible() &&
         layer.showAtZoomLevel(zoom) &&
         !layer.hasErrors() &&
-        !layer.isDataLoaded()
+        !layer.isInitialDataLoadComplete()
```

Comment:
Would it also make sense to rename `areLayersLoaded`?

Reply:
discussed offline. Keeping name, because it wraps some more checks such as visiblity and errors.